src/option/option-window.tsx

```diff
@@ -60,6 +63,22 @@ export default class OptionWindow extends React.Component {
 		});
 	}
 
+	UnRequireInput = () => {
+		const author = document.getElementById('authorInput') as HTMLInputElement;
+		const flag = document.getElementById('flagInput') as HTMLInputElement;
+		if (!author.value) {
+			flag.required = true;
+		} else {
+			flag.required = false;
+		}
```

Comment:
`flag.required = !author.value`; similarly below.

Reply:
will apply in the next PR